nuxt.config.ts

```diff
@@ -1,12 +1,10 @@
-import '@vite-pwa/nuxt';
-import 'nuxt';
```

Comment:
The imports are there because sometimes TS is being stupid about the types, especially before you run `yarn dev` once. Please restore them.

Reply:
I personally didn't need them, and we can't just load all the modules we load with Nuxt.